### DISCUSSION TITLE
[review] BackTop 组件

### DISCUSSION THREAD

**Discussion by yangxiaolu1993** [jdf2e/nutui-react#9] [General]
1、 是否有动画,和duration参数互斥是什么意思？

   `isAnimation && duration > 0 ? scrollAnimation() : scroll()`
   
2、不能自定义 className 与 style
    `<BackTop
            elId="elId"
            distance={200}
            className='backbox'
            backTopClick={() => {
              console.log('触发返回顶部')
            }}
          > `
3、 prop：isAnimation 不生效

4、组件内定义的 className 值未使用 bem 函数

**Comment by vickyYE** [comment]
1、动画和duration没有互斥，在duration大于0时动画才生效
其他问题已修改
